mono/metadata/threads.c

```diff
@@ -4485,24 +4524,36 @@ static void CALLBACK dummy_apc (ULONG_PTR param)
 #endif
 
 /*
- * mono_thread_execute_interruption
+ * mono_thread_execute_interruption_handle
  * 
  * Performs the operation that the requested thread state requires (abort,
  * suspend or stop)
  */
-static MonoException*
-mono_thread_execute_interruption (void)
+static MonoExceptionHandle
+mono_thread_execute_interruption_handle (void)
 {
-	MonoInternalThread *thread = mono_thread_internal_current ();
-	MonoThread *sys_thread = mono_thread_current ();
+// FIXME This function does unnecessary allocations and handle indirections.
```

Comment:
Fix indentation

Reply:
Do FIXME comments deserve proper indentation, or are they bugs and make them slightly worse? And give comments sometimes more horizontal space.

Reply:
This is a comment inside a function, and so should be indented like every other piece of code in a function. This is a comment that can span mutiple lines, like every other piece of code, so if it gets to long, simply break the line. In short, please follow coding convention.

Reply:
It is little non obvious where enter/exit is needed, and it does appear needed here. It is callstack dependent, as I understand, you have to look up the static callgraph and decide if someone will have entered/exited, and not be looping. In this case there does not appear to be an enter/exit in caller.